### ISSUE TITLE
MODX 3 lexicon->load() crashes $_SESSION not set

### ISSUE THREAD

**Issue by BobRay** [modxcms/revolution#15373]
### Summary
I no $_SESSION array is set, lexicon->load crashes.

### Step to reproduce
Call lexicon->load() with no $_SESSION set and $modx->context set to 'mgr'.

Here is the offending code (around line 189 of core/src/Revolution/modLexicon.php)

```
if ($this->modx->context && $this->modx->context->get('key') == 'mgr') {
            $defaultLanguage = $this->modx->getOption('manager_language', $_SESSION,
                $this->modx->getOption('cultureKey', null, 'en'));
        } else {
            $defaultLanguage = $this->modx->getOption('cultureKey', null, 'en');
        }
```

This will break testing code. (i.e. Unit, Integration, or Acceptance test) that call `$lexicon->load();`. 

There are a number ways to fix this. This is one (just above the first getOption() call), but I'm not sure it's the best:

```
if (! isset($_SESSION)) {
    $_SESSION = null;
}
```

### Environment
MODX 3, Windows 10,  apache

**Comment by Mark-H** [comment]
What do you mean exactly by "crashes"? Is it causing an error or unexpected behavior downstream?

**Comment by BobRay** [comment]
Sorry, attempting to access a member of an array that doesn't exist throws a PHP error: Unknown variable `$_SESSION`.

**Comment by BobRay** [comment]
I think a better solution is to put this above the first getOption() call:

```
$_SESSION = isset($_SESSION)? $_SESSION : array();
```

**Comment by Ibochkarev** [comment]
Fixed #15377
